test/workers/test_virtual.py

```diff
@@ -41,10 +41,12 @@ def test_send_msg():
     me.send_msg(ObjectMessage(obj), bob)
     elapsed_time = time() - start_time
 
+    me.message_pending_time = 0
```

Comment:
Q: There is still a problem with this approach if any of the above lines will throw an exception, then the value for ```message_pending_time`` remains 0

Reply:
I agree, a python context would be better here but let's say it's ok for now